src/App.js

```diff
@@ -1,40 +1,274 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+import React,  { useState, useEffect } from "react";
 import "./App.css";
-import BottomRow from "./BottomRow";
+import BottomRow from "./Components/BottomRow";
+import ToGoForm from "./Components/ToGoForm";
+import BallOnForm from "./Components/BallOnForm";
+import SetTimeForm from "./Components/SetTimeForm";
+import HomeNameForm from "./Components/HomeNameForm";
+import AwayNameForm from "./Components/AwayNameForm";
+import HomeScoreForm from "./Components/HomeScoreForm";
+import AwayScoreForm from "./Components/AwayScoreForm";
+import SetHomeTimeoutsForm from "./Components/SetHomeTimeoutsForm";
+import SetAwayTimeoutsForm from "./Components/SetAwayTimeoutsForm";
+
 
 function App() {
-  //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
+  //TODO: STEP 2 - Establish your application's state with some useState hooks.  You'll need one for the home score and another for the away score.
+  const [homeScore, setHomeScore] = useState(0);
+  const [awayScore, setAwayScore] = useState(0);
+  const [down, setDown] = useState(1);
+  const [quarter, setQuarter] = useState(1);
+  const [toGo, setToGo] = useState(10);
+  const [toGoStatus, setToGoStatus] = useState(false);
+  const [ballOn, setBallOn] = useState(35);
+  const [ballOnStatus, setBallOnStatus] = useState(false);
+  const [time, setTime] = useState(900);
+  const [timeStatus, setTimeStatus] = useState(false);
+  const [manualTimeStatus, setManualTimeStatus] = useState(false);
+  const [homeName, setHomeName] = useState("Home");
+  const [homeNameStatus, setHomeNameStatus] = useState(false);
+  const [homeScoreStatus, setHomeScoreStatus] = useState(false);
+  const [awayName, setAwayName] = useState("Away");
+  const [awayNameStatus, setAwayNameStatus] = useState(false);
+  const [awayScoreStatus, setAwayScoreStatus] = useState(false);
+  const [homeTimeouts, setHomeTimeouts] = useState(3);
+  const [awayTimeouts, setAwayTimeouts] = useState(3);
+  const [homeTimeoutsStatus, setHomeTimeoutsStatus] = useState(false);
+  const [awayTimeoutsStatus, setAwayTimeoutsStatus] = useState(false);
+
+  
+  useEffect(() => {
+    let interval = null;
+    if (timeStatus) {
+      interval = setInterval(() => {
```

Comment:
How did you know how to useEffect here? You even set up un-mounting below. This is great work.